src/api-utils/responseSchemas/gamedev.ts

```diff
@@ -2,6 +2,38 @@ import Joi from 'joi';
 
 import { ScoreStrategy, ResetStrategy } from '../../models/LeaderboardEntry';
 
+export const postLeaderboardResultScoreSchema = Joi.object({
+  id: Joi.number().optional(),
+  _leaderboardEntryId: Joi.number().required(),
+  _userId: Joi.number().required(),
+  score: Joi.number().required(),
+  _leaderboardResultsMeta: Joi.array()
+    .items(
+      Joi.object({
+        attribute: Joi.string().required(),
+        value: Joi.string().required(),
+      })
+    )
+    .optional(),
+}).required();
+
+export const leaderboardResultSchema = Joi.object({
+  id: Joi.number().required(),
+  _leaderboardEntryId: Joi.number().required(),
+  _userId: Joi.number().required(),
+  score: Joi.number().required(),
+  _leaderboardResultsMeta: Joi.array()
+    .items(
+      Joi.object({
+        attribute: Joi.string().required(),
+        value: Joi.string().required(),
+      })
+    )
+    .optional(),
+  createdAt: Joi.date(),
+  updatedAt: Joi.date(),
+}).required();
+
```

Comment:
I have the feeling that this file will become pretty in the future.. we should probably start thinking in spliting it